packages/mesos/extra/dcos-mesos-agent.service

```diff
@@ -14,7 +14,9 @@ EnvironmentFile=-/opt/mesosphere/etc/mesos-slave-common-extras
 EnvironmentFile=/opt/mesosphere/etc/mesos-slave
 EnvironmentFile=-/var/lib/dcos/mesos-slave-common
 EnvironmentFile=/var/lib/dcos/mesos-resources
+EnvironmentFile=-/run/dcos/etc/mesos-slave
```

Comment:
Can we please do `/run/dcos/<service-name>/<whatever the service wants>`? This current layout makes making all of this stuff non-root next to impossible without moving everything.